sdk/data/synths.ts

```diff
@@ -40,15 +34,6 @@ type BasicSynth = {
 };
 
 export const synths: Record<SynthSymbol, BasicSynth> = {
-	sAAVE: {
-		name: 'Aave',
-		asset: 'AAVE',
-		addresses: {
-			1: '0xd2dF355C19471c8bd7D8A3aa27Ff4e26A21b4076',
```

Comment:
We should keep Aave on L1 Ethereum (chainID 1) as it seems to be only deprecated on L2 (chainID 10) so far.